--- oiio-Release-1.2.0/src/libOpenImageIO/CMakeLists.txt~	2013-07-15 22:08:32.907370256 +0800
+++ oiio-Release-1.2.0/src/libOpenImageIO/CMakeLists.txt	2013-07-15 22:09:31.859060898 +0800
@@ -208,7 +208,7 @@
 else ()
     add_library (OpenImageIO SHARED ${libOpenImageIO_srcs})
 endif ()
-target_link_libraries (OpenImageIO
+target_link_libraries (OpenImageIO ${CMAKE_DL_LIBS}
                            ${VISIBILITY_COMMAND} ${VISIBILITY_MAP_COMMAND}
                            ${Boost_LIBRARIES})
 
--- ./src/libutil/CMakeLists.txt.orig	2014-09-26 05:54:15.480626247 +0100
+++ ./src/libutil/CMakeLists.txt	2014-09-26 05:54:27.506625361 +0100
@@ -9,7 +9,7 @@
 else ()
     add_library (OpenImageIO_Util SHARED ${libOpenImageIO_Util_srcs})
 endif ()
-target_link_libraries (OpenImageIO_Util
+target_link_libraries (OpenImageIO_Util ${CMAKE_DL_LIBS}
                            ${VISIBILITY_COMMAND} ${VISIBILITY_MAP_COMMAND}
                            ${Boost_LIBRARIES})
 if (WIN32)
