--- oiio-Release-1.7.14/src/libOpenImageIO/CMakeLists.txt.0000~	2017-05-13 14:00:51.537576735 +0200
+++ oiio-Release-1.7.14/src/libOpenImageIO/CMakeLists.txt	2017-05-13 14:11:14.904015803 +0200
@@ -97,7 +97,7 @@ if (BUILDSTATIC)
 else ()
     add_library (OpenImageIO SHARED ${libOpenImageIO_srcs})
 endif ()
-target_link_libraries (OpenImageIO
+target_link_libraries (OpenImageIO ${CMAKE_DL_LIBS}
                        ${VISIBILITY_COMMAND} ${VISIBILITY_MAP_COMMAND}
                        ${format_plugin_libs} # Add all the target link libraries from the plugins
                        ${Boost_LIBRARIES})
--- oiio-Release-1.7.14/src/libutil/CMakeLists.txt.0000~	2017-05-01 19:21:17.000000000 +0200
+++ oiio-Release-1.7.14/src/libutil/CMakeLists.txt	2017-05-13 14:00:51.538576743 +0200
@@ -9,7 +9,7 @@ if (BUILDSTATIC)
 else ()
     add_library (OpenImageIO_Util SHARED ${libOpenImageIO_Util_srcs})
 endif ()
-target_link_libraries (OpenImageIO_Util
+target_link_libraries (OpenImageIO_Util ${CMAKE_DL_LIBS}
                            ${VISIBILITY_COMMAND} ${VISIBILITY_MAP_COMMAND}
                            ${Boost_LIBRARIES})
 if (WIN32)
