--- OpenImageIO-oiio-388f38b/src/libOpenImageIO/CMakeLists.txt.dl~	2012-03-19 18:01:39.000000000 +0100
+++ OpenImageIO-oiio-388f38b/src/libOpenImageIO/CMakeLists.txt	2012-04-02 11:27:32.143599273 +0200
@@ -197,7 +197,7 @@ if (BUILDSTATIC)
 else ()
     add_library (OpenImageIO SHARED ${libOpenImageIO_srcs})
 endif ()
-target_link_libraries (OpenImageIO ${Boost_LIBRARIES})
+target_link_libraries (OpenImageIO ${Boost_LIBRARIES} ${CMAKE_DL_LIBS})
 
 
 
