--- oiio-Release-1.8.9/src/libutil/CMakeLists.txt.orig	2018-03-04 00:07:13.677965263 +0000
+++ oiio-Release-1.8.9/src/libutil/CMakeLists.txt	2018-03-04 00:08:09.370986184 +0000
@@ -16,6 +16,7 @@
     target_link_libraries (OpenImageIO_Util psapi.lib)
 endif ()
 target_link_libraries (OpenImageIO_Util ${ILMBASE_LIBRARIES})
+target_link_libraries (OpenImageIO_Util ${CMAKE_DL_LIBS})
 
 if (VERBOSE)
     message(STATUS "Setting SOVERSION to: ${SOVERSION}")
--- oiio-Release-1.8.9/src/libOpenImageIO/CMakeLists.txt.orig	2018-03-04 00:06:59.182480475 +0000
+++ oiio-Release-1.8.9/src/libOpenImageIO/CMakeLists.txt	2018-03-04 00:08:53.353708965 +0000
@@ -105,7 +105,7 @@
 #     target_link_libraries (OpenImageIO ${SANITIZE_LIBRARIES})
 # endif ()
 
-target_link_libraries (OpenImageIO
+target_link_libraries (OpenImageIO ${CMAKE_DL_LIBS}
                        ${SANITIZE_LIBRARIES}
                        ${format_plugin_libs} # Add all the target link libraries from the plugins
                        ${Boost_LIBRARIES}
